service-commands/scripts/find-alembic-changes.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# finds all revisions that contains mention of a certain table, field, etc
```

Comment:
I was originally just looking at the alembics/ but couldn't figure out how best to extract ordering.

Is there perhaps an alembic command that does this already?

Reply:
if you want to find all references to a revision, i don't think it shows that but usually between `alembic current` and `alembic history` it should show you everything you generally need

Reply:
Indeed, the script makes use of `alembic history` and `alembic show` to show ordered and grepped schema information.

fwiw, my `alembic current` output is blank.

Reply:
@joaquincasares just curious how you use this

Reply:
This is some output:

```
❯ ../service-commands/scripts/find-alembic-changes.sh trending_params
dc7f691adc79
3ea8467836d8
8e4dda8255fd
a6d2e50a8efa DROP MATERIALIZED VIEW IF EXISTS trending_params_aSPET;
7104383ac0fe
f775fb87f5ff
be27a2794f75 # This revision copies 6b5186e7d28f and replaces trending_params with a fixed version ALTER MATERIALIZED VIEW trending_params RENAME TO trending_params_aSPET; ALTER INDEX trending_params_track_id_idx RENAME TO trending_params_track_id_idx_aSPET; CREATE MATERIALIZED VIEW trending_params as CREATE INDEX trending_params_track_id_idx ON trending_params (track_id); DROP MATERIALIZED VIEW trending_params; ALTER MATERIALIZED VIEW trending_params_aSPET RENAME to trending_params;
85680793a611
132955202f03
a41727360622
75b670a37598
6b5186e7d28f DROP MATERIALIZED VIEW IF EXISTS trending_params; -- trending_params from 92571f94989a CREATE MATERIALIZED VIEW trending_params as CREATE INDEX trending_params_track_id_idx ON trending_params (track_id); DROP MATERIALIZED VIEW IF EXISTS trending_params; -- trending_params from 92571f94989a CREATE MATERIALIZED VIEW trending_params as CREATE INDEX trending_params_track_id_idx ON trending_params (track_id);
36eac5ed00bf
591a66d44e5b
edccccc274a7
8bc5bac6711b
e2a8aea2e2e1 DROP MATERIALIZED VIEW IF EXISTS trending_params; -- trending_params from 92571f94989a CREATE MATERIALIZED VIEW trending_params as CREATE INDEX trending_params_track_id_idx ON trending_params (track_id); DROP MATERIALIZED VIEW IF EXISTS trending_params; -- trending_params from 92571f94989a CREATE MATERIALIZED VIEW trending_params as CREATE INDEX trending_params_track_id_idx ON trending_params (track_id);
7857f02c8438
92571f94989a DROP MATERIALIZED VIEW IF EXISTS trending_params; CREATE MATERIALIZED VIEW trending_params as CREATE INDEX trending_params_track_id_idx ON trending_params (track_id); DROP INDEX IF EXISTS trending_params_track_id_idx; DROP MATERIALIZED VIEW trending_params;
65ec7a3171c7
561ba1eca13a
cbf0fafabdeb
7693ab16e2e1
169519f975c2
b40b074a75be
9ff8c8ae8de2
badd3aab7e5e
9562cf365cf4
f64a484f1496
ed974e76d415
cb9aa46f1e46
273b8bcef694
2e02a681aeaa
534987cb0355
6cf091d52869
...
```

Since `trending_params` was dropped via `DROP MATERIALIZED VIEW IF EXISTS aggregate_track CASCADE;`, I needed to recreate `trending_params` in order to reference the new `TABLE aggregate_track`.

This script allowed me to find the most recent schema related to `trending_params` that would require recreation in my alembic migration, as seen in my migrations `CASCADING_SCHEMA`.

Reply:
this is actually pretty cool